File: app/src/main/java/com/kode/utair/presentation/ui/activities/BaseMvpActivity.java
package com.kode.utair.presentation.ui.activities;

import android.content.Context;

import com.arellomobile.mvp.MvpAppCompatActivity;

import uk.co.chrisjenx.calligraphy.CalligraphyContextWrapper;

public class BaseMvpActivity extends MvpAppCompatActivity {

    @Override
    protected void attachBaseContext(Context newBase) {
        super.attachBaseContext(CalligraphyContextWrapper.wrap(newBase));
    }

}

File: app/src/main/java/com/kode/utair/presentation/ui/activities/BaseMvpActivity.kt
package com.kode.utair.presentation.ui.activities

import android.content.Context
import com.kode.utair.presentation.ui.base.MvpAppCompatActivity
import io.github.inflationx.viewpump.ViewPumpContextWrapper

open class BaseMvpActivity : MvpAppCompatActivity() {

    override fun attachBaseContext(newBase: Context) {
        super.attachBaseContext(ViewPumpContextWrapper.wrap(newBase))
    }

}